src/components/dialogs/csv-import-filter-creation-dialog.js

```diff
@@ -59,13 +59,14 @@ const CsvImportFilterCreationDialog = ({
 
     const csvData = () => {
         let newData = [...data];
-        // Adds 10 empty lines in the CSV file before adding the comment.
-        for (let i = 0; i < 10; i++) {
-            newData.push([]);
+        if (formType === ElementType.CONTINGENCY_LIST) {
```

Comment:
You don't want to keep some empty rows before the comments ?
It's easier for the user not having to insert new rows

Reply:
Why can't the user insert his data bellow the comments ? Anyway, I'll see with the product owner.